app/Config/Routes.php

```diff
@@ -26,3 +26,7 @@
 $routes->get('health', [HealthCheck::class, 'check']);
 $routes->get('globals', [Globals::class, 'get']);
 $routes->put('dashboard/admin/globals', [AdminDashboard::class, 'setGlobals'], ['filter' => AdminAuthFilter::class]);
+$routes->get('dashboard/admin/all-events', [AdminDashboard::class, 'getAllEvents'], ['filter' => AdminAuthFilter::class]);
```

Comment:
When this route returns all fields of all events do we even need a specific route then?
I thought that I would have to make 2 requests.
One that is returning the ids and event years(or something else to display in a dropdown) and one that returns the data of the currently visible year.
But in this case I have already all data of all events. So one request would be enough. Right?
But the request can get quite big over time.

What do you think?